tests/python/contrib/test_ethosu/test_codegen.py

```diff
@@ -902,6 +929,7 @@ def depthwise_conv2d(x):
 @pytest.mark.xfail(strict=False, reason="See https://github.com/apache/tvm/issues/10487")
```

Comment:
I think this solves this issue according to our offline discussion.

Reply:
I think we should do this in a follow up incase this isn't the reason for the failure, WDYT?

Reply:
ok. sounds good to me.